### DISCUSSION TITLE
Making an entity invisible with EntityMetaData packet

### DISCUSSION THREAD

**Discussion by ghost** [Minestom/Minestom#1953] [Help] [answered]
I have an armor stand that was spawned for a specific player with SpawnEntityPacket, and I'm attempting to make it invisible. How would I go about this using packets?

**Comment by MrGazdag** [comment, marked as answer]
With Minestom's viewable rules, i don't see why you would want to simulate entities exclusively with packets. You can set who can view entities with the viewable rules, so you can use this to create per-player entities.

**Reply by ghost** [reply]
Never knew about that, thanks a lot!
